lib/gui/etcher.ts

```diff
@@ -53,13 +56,21 @@ async function checkForUpdates(interval: number) {
 					packageUpdated = true;
 				}
 			} catch (err) {
-				logException(err);
+				logMainProcessException(err);
 			}
 		}
 		await delay(interval);
 	}
 }
 
+function logMainProcessException(error: any) {
+	const shouldReportErrors = settings.getSync('errorReporting');
+	if (shouldReportErrors) {
+		console.error(error);
+		SentryMain.captureException(error);
+	}
+}
```

Comment:
Shouldn't the `console.error` be _outside_ the `shouldReportErrors` check, because the user still wants to see errors in the DevConsole, even if they've turned off reporting errors to Balena?

Reply:
Totally! I'll update this in next PR.

Reply:
made it part of this PR : https://github.com/balena-io/etcher/pull/3978/commits